docs/julia.rst

```diff
@@ -328,15 +336,17 @@ The following functions must be available:
 +---------------------------------------------------------------------------------------------------------------+
 | mulmodu256(x:u256, y:u256, m:u256) -> z:u256| (x * y) % m with arbitrary precision arithmetics                |
 +---------------------------------------------------------------------------------------------------------------+
-| ltu256(x:u256, y:u256) -> z:bool            | 1 if x < y, 0 otherwise                                         |
+| ltu256(x:u256, y:u256) -> z:bool            | true if x < y, false otherwise                                  |
++---------------------------------------------------------------------------------------------------------------+
+| gtu256(x:u256, y:u256) -> z:bool            | true if x > y, false otherwise                                  |
 +---------------------------------------------------------------------------------------------------------------+
-| gtu256(x:u256, y:u256) -> z:bool            | 1 if x > y, 0 otherwise                                         |
+| sltu256(x:s256, y:s256) -> z:bool           | true if x < y, false otherwise, for signed numbers in two's complement |
```

Comment:
Can you re-align the `|`? (it is fine to break the line, it will not result in an explicit line break in the rendered version.

Reply:
Not only that, some other parts are broken, wanted to finish that in this PR hence it is WIP.